build/Autofac.Build.psm1

```diff
@@ -208,7 +219,14 @@ function Invoke-Test
     {
       Push-Location $Project
 
-      & dotnet test --configuration Release --logger:trx
+    & dotnet test `
```

Comment:
You probably want to include any generated code. I use these arguments:

```
/p:ExcludeByAttribute=CompilerGeneratedAttribute /p:ExcludeByAttribute=GeneratedCodeAttribute
```

Reply:
You meant exclude rather than include, right? Then I've addressed that. :+1:

Reply:
Ah, yes, exclude... 😅